Common/Packets/BacktestNodePacket.cs

```diff
@@ -86,6 +86,11 @@ public class BacktestNodePacket : AlgorithmNodePacket
         /// </summary>
         public bool IsDebugging => Breakpoints.Any();
 
+        /// <summary>
+        /// Optional initial cash amount if set
+        /// </summary>
+        [JsonProperty(PropertyName = "decCashAmount")]
```

Comment:
Was trying to find name convention for assigning Json property. Could not find one. Correction welcome

Reply:
I think for improved future proof, we should use a `List<CashAmount>`, each `CashAmount` instance holds a value and a currency. Regarding the JSON property name, it seems to me it should be `aCashAmounts`

Reply:
If the aim is to support multiple currencies, I would lean towards a dictionary instead of a list to enforce uniqueness.